guides/contributor/README.md

```diff
@@ -0,0 +1,66 @@
+# OpenTelemetry New Contributor Guide
```

Comment:
Should this replace or add to https://github.com/open-telemetry/community/blob/main/CONTRIBUTING.md?

Reply:
I was thinking we migrate the content from there over into this, and then expand it?

Reply:
works for me, I just wanted to verify that we do not have 2 documents at the end.